tests/ci_tests/full_test.py

```diff
@@ -38,4 +41,31 @@ def iter_function(epoch, error, test_error, test_error_gradient, test_accuracy):
 
 predictor.learn(from_data=df, callback_on_iter=iter_function, eval_every_x_epochs=2, stop_training_after_seconds=120)
 
-print(predictor.predict(when={'number_of_rooms':3, 'number_of_bathrooms':2, 'sqft':700, 'location':'great'}))
+#print(predictor.predict(when={ 'number_of_bathrooms':2, 'sqft':700, 'location':'great'}))
+
+predictor.save('/tmp/test.lw')
```

Comment:
oh, you have to use `os.join` to build the path here else it fails on windows, see the travis error